specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -63,7 +63,7 @@ We will prioritize the following feature set for Windows App SDK:
 A Portal Registration flow through
 [AAD (Azure Active Directory)](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app)
 will also be defined that removes the dependency of the Push Flow with the Partner Center Portal.
-The RemoteIdentifier GUID in this spec maps to the ApplicationId in the AAD App Registration
+The RemoteIdentifier GUID in this spec maps to the ObjectId in the AAD App Registration
```

Comment:
This looks simple enough to me, but please update the the [PushNotifications quickstart page](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push-notifications/push-quickstart) with all references to "remoteId".